cob_bringup/robots/raw3-5.xml

```diff
@@ -39,29 +39,34 @@
 			<arg name="name" value="base_laser_rear"/>
 			<arg name="sim" value="$(arg sim)"/>
 		</include>
-		<include file="$(find cob_bringup)/components/legacy_base.launch">
-			<arg name="robot" value="$(arg robot)"/>
-			<arg name="sim" value="$(arg sim)"/>
-		</include>
 
-		<!-- start common nodes used in hardware and simulation -->
-		<include file="$(find cob_bringup)/drivers/scan_unifier.launch">
-			<arg name="robot" value="$(arg robot)"/>
+		<include file="$(find cob_bringup)/tools/emergency_stop_monitor.launch" >
+			<arg name="robot" value="$(arg robot)" />
 		</include>
-		<include file="$(find cob_bringup)/tools/emergency_stop_monitor.launch">
-			<arg name="robot" value="$(arg robot)"/>
+		<include file="$(find cob_bringup)/drivers/battery_voltage.launch" >
+			<arg name="robot" value="$(arg robot)" />
 		</include>
-		<include file="$(find cob_bringup)/drivers/battery_voltage.launch">
-			<arg name="robot" value="$(arg robot)"/>
+		<!-- problems when using battery_filter
+		<include file="$(find cob_bringup)/drivers/battery_voltage_filter.launch" >
+			<arg name="robot" value="$(arg robot)" />
 		</include>
-		<include file="$(find cob_bringup)/drivers/battery_voltage_filter.launch">
-			<arg name="robot" value="$(arg robot)"/>
+		-->
+		<include file="$(find cob_bringup)/tools/battery_monitor.launch" >
+			<arg name="robot" value="$(arg robot)" />
 		</include>
-		<!--<include file="$(find cob_bringup)/tools/battery_monitor.launch">
-			<arg name="robot" value="$(arg robot)"/>
-		</include>-->
-		<include file="$(find cob_bringup)/tools/diagnostics_aggregator.launch">
-			<arg name="robot" value="$(arg robot)"/>
+		<include file="$(find cob_bringup)/components/canopen_base.launch" >
+			<arg name="robot" value="$(arg robot)" />
+			<arg name="component_name" value="base" />
+			<arg name="can_device" value="can1" />
+			<arg name="interval_ms" value="20" />
+			<arg name="use_external_sync" value="true" />
```

Comment:
external sync was not tested properly for use with all components.
It will get the default somewhen (in `canopen_base.launch`)